nil pointer dereference using -disable_metrics=disk

The code doesn't check for self.fsHandler == nil, which it is when disk metrics are disabled

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x20 pc=0x65a801]

goroutine 35 [running]:
github.com/google/cadvisor/container/docker.(*dockerContainerHandler).Start(0xc2080f67e0)
	/root/cadvisor/src/github.com/google/cadvisor/container/docker/handler.go:204 +0x41
github.com/google/cadvisor/manager.(*containerData).housekeeping(0xc20814d4a0)
	/root/cadvisor/src/github.com/google/cadvisor/manager/container.go:377 +0x58
created by github.com/google/cadvisor/manager.(*containerData).Start
	/root/cadvisor/src/github.com/google/cadvisor/manager/container.go:94 +0x44

--- a/container/docker/handler.go
+++ b/container/docker/handler.go
@@ -201,11 +201,15 @@ func newDockerContainerHandler(
 
 func (self *dockerContainerHandler) Start() {
 	// Start the filesystem handler.
-	self.fsHandler.start()
+	if self.fsHandler != nil {
+		self.fsHandler.start()
+	}
 }
 
 func (self *dockerContainerHandler) Cleanup() {
-	self.fsHandler.stop()
+	if self.fsHandler != nil {
+		self.fsHandler.stop()
+	}
 }
 
 func (self *dockerContainerHandler) ContainerReference() (info.ContainerReference, error) {

